app/src/main/java/com/iron/espresso/presentation/home/study/adapter/viewholder/PlaceViewHolder.kt

```diff
@@ -0,0 +1,27 @@
+package com.iron.espresso.presentation.home.study.adapter.viewholder
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.recyclerview.widget.RecyclerView
+import com.iron.espresso.R
+import com.iron.espresso.model.response.Place
+
+class PlaceViewHolder(parent: ViewGroup) : RecyclerView.ViewHolder(
+    LayoutInflater.from(parent.context).inflate(R.layout.item_place_list, parent, false)
+) {
+    private val binding =
+        DataBindingUtil.bind<com.iron.espresso.databinding.ItemPlaceListBinding>(itemView)
+
+    fun bind(item: Place, itemClickListener: (title: String) -> Unit) {
```

Comment:
```suggestion
    fun bind(item: Place) {
```
onClickListener를 매번 달아줄 필요가 있을까요? 
일반적으로 보통의 뷰에 한번만 달아주는 동작인데요.

Reply:
맞는 의견같아요..이부분은 저도 생각해봐서 반영하도록 해볼게요.

Reply:
590717b